labgrid/driver/rawnetworkinterfacedriver.py

```diff
@@ -0,0 +1,94 @@
+# pylint: disable=no-member
+import json
+import signal
+import subprocess
+import shutil
+import tempfile
+
+import attr
+
+from .common import Driver
+from ..factory import target_factory
+from ..step import step
+from ..util.helper import processwrapper
+from ..util.managedfile import ManagedFile
+from ..resource.common import NetworkResource
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class RawNetworkInterfaceDriver(Driver):
+    bindings = {
+        "iface": {"NetworkInterface", "RemoteNetworkInterface", "USBNetworkInterface"},
+    }
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+        self.proc_pcap_map = {}
```

Comment:
This should not be needed as we have one Driver per Interface.

Instead remember one process per record and capture for consistency checks on start/stop.

Reply:
Done.